inst/gams/scripts/declarations.gms

```diff
@@ -51,10 +51,10 @@ p_calibDeviationRen(iteration,bs,hs,bsr,hsr,vin,reg,loc,typ,inc,ttot) "Ratio of
 scalars
 t0 "reference year for discounting"
 
-epsilon "offset to avoid log(0)" /1E-4/
+epsilon "offset to avoid log(0)" /1E-5/
 
-priceSensBS "price sensitivity of building shell choice" /1E-1/
-priceSensHS "price sensitivity of heating system choice" /2E-1/
+priceSensBS "price sensitivity of building shell choice" /5E-2/
+priceSensHS "price sensitivity of heating system choice" /1E-1/
```

Comment:
Why did you change these parameters?

Reply:
This was part of the manual calibration. These values should be set rigorously in the calibration one day. I try to find a value that minimises deviation from the target values before setting any inconvenience costs and then use the ladder to fine tune.